docker/pg.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM postgres:15
+
+# Variables needed at runtime to configure postgres and run the initdb scripts
+ENV POSTGRES_DB 'postgres'
+ENV POSTGRES_USER 'postgres'
+ENV POSTGRES_PASSWORD 'postgres'
```

Comment:
How serious is the pg dockerfile? Should we hide its credentials similar to the above,
`POSTGRES_PASSWORD: ${PGPASSWORD:-postgres}`?